### PR TITLE
[Image Component] image sizes property becomes 100vw by default

### PR DESCRIPTION
**Pull request by bmvantunes**
Currently if sizes is not defined, Next.js is setting sizes as:
```
(max-width: 640px) 640px, (max-width: 750px) 750px, (max-width: 828px) 828px, (max-width: 1080px) 1080px, (max-width: 1200px) 1200px, (max-width: 1920px) 1920px, (max-width: 2048px) 2048px, 3840px'
```

This pull request will make sizes be `100vw` by default, which will allow us to download "smaller" images than what's currently happening. 

In a demo app I have, the difference is between downloading 488KB vs 1.4MB (in images)

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| buildDuration | 11s | 10.9s | -164ms |
| nodeModulesSize | 82.5 MB | 82.5 MB | -311 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.668 | 2.677 | ⚠️ +0.01 |
| / avg req/sec | 937 | 933.85 | ⚠️ -3.15 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.593 | 1.655 | ⚠️ +0.06 |
| /error-in-render avg req/sec | 1569.59 | 1510.44 | ⚠️ -59.15 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| 677f882d2ed8..34b9.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3e8cb31..cddc.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-01375..90b6.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..fd29.js gzip | 303 B | 303 B | ✓ |
| withRouter-b..76ae.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| buildDuration | 13s | 12.9s | -47ms |
| nodeModulesSize | 82.5 MB | 82.5 MB | -311 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| 677f882d2ed8..34b9.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3e8cb31..cddc.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-01375..90b6.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..fd29.js gzip | 303 B | 303 B | ✓ |
| withRouter-b..76ae.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.16 MB | 5.16 MB | ✓ |

</details>

</details>
Commit: b9f3ad8b0df5c358fe3248151938453c7f959fd6

File: docs/api-reference/next/image.md [hunk 1]
```diff
@@ -93,7 +93,9 @@ Try it out:
 
 ### sizes
 
-A string mapping media queries to device sizes. Defaults to [Device Sizes](/docs/basic-features/image-optimization.md#device-sizes).
+A string mapping media queries to device sizes. Defaults to `100vw`.
+
+We recommend setting `sizes`, as it helps the browser choose the right image size to use.
```
Comment (styfle) [inline docs/api-reference/next/image.md:98]:
```suggestion
We recommend setting `sizes` when `layout="responsive"` and your image will not be the same width as the viewport.
```

**Review by styfle — APPROVED** [review]
Looks good, great job! 🎉 🥇 🥳

**Comment by bmvantunes** [comment]
> Looks good, great job! 🎉 🥇 🥳

Thank you Steven =D

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| buildDuration | 8.4s | 8.6s | ⚠️ +174ms |
| nodeModulesSize | 82.5 MB | 82.5 MB | -311 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.069 | 2.076 | ⚠️ +0.01 |
| / avg req/sec | 1208.43 | 1204.26 | ⚠️ -4.17 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.107 | 1.087 | -0.02 |
| /error-in-render avg req/sec | 2257.67 | 2300.03 | +42.36 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| 677f882d2ed8..34b9.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3e8cb31..cddc.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-01375..90b6.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..fd29.js gzip | 303 B | 303 B | ✓ |
| withRouter-b..76ae.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| buildDuration | 10.2s | 10.2s | -5ms |
| nodeModulesSize | 82.5 MB | 82.5 MB | -311 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| 677f882d2ed8..34b9.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3e8cb31..cddc.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-01375..90b6.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..fd29.js gzip | 303 B | 303 B | ✓ |
| withRouter-b..76ae.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.16 MB | 5.16 MB | ✓ |

</details>

</details>
Commit: 5944f6994e69e79b56836dc5c6e9de856ae54063

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| buildDuration | 8.8s | 8.7s | -159ms |
| nodeModulesSize | 82.5 MB | 82.5 MB | -311 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.152 | 2.137 | -0.02 |
| / avg req/sec | 1161.61 | 1169.7 | +8.09 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.135 | 1.145 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 2203.36 | 2182.5 | ⚠️ -20.86 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| 677f882d2ed8..34b9.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3e8cb31..cddc.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-01375..90b6.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..fd29.js gzip | 303 B | 303 B | ✓ |
| withRouter-b..76ae.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| buildDuration | 10.4s | 10.4s | -69ms |
| nodeModulesSize | 82.5 MB | 82.5 MB | -311 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| 677f882d2ed8..34b9.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3e8cb31..cddc.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-01375..90b6.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..fd29.js gzip | 303 B | 303 B | ✓ |
| withRouter-b..76ae.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.16 MB | 5.16 MB | ✓ |

</details>

</details>
Commit: 5a0028609ec11d26d6015777c61f460e8b8644b3

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 5a0028609ec11d26d6015777c61f460e8b8644b3

**test/integration/production-build-dir/test/index.test.js** 
- Production Custom Build Directory > With basic usage > should render the page

<details>
<summary>Expand output</summary>

  ● Production Custom Build Directory › With basic usage › should render the page

    expect(received).toBe(expected) // Object.is equality

    Expected: ""
    Received: "The postprocess middleware \"Inline-Fonts\" took 11ms(11, 0) to complete. This is longer than the 10 limit.
    "

      20 |         stderr: true,
      21 |       })
    > 22 |       expect(result.stderr).toBe('')
         |                             ^
      23 | 
      24 |       const app = nextServer({
      25 |         dir: join(__dirname, '../build'),

      at Object.<anonymous> (integration/production-build-dir/test/index.test.js:22:29)

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| buildDuration | 8.5s | 8.5s | -9ms |
| nodeModulesSize | 82.5 MB | 82.5 MB | -311 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.062 | 2.066 | 0 |
| / avg req/sec | 1212.29 | 1209.79 | ⚠️ -2.5 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.049 | 1.092 | ⚠️ +0.04 |
| /error-in-render avg req/sec | 2384.2 | 2288.87 | ⚠️ -95.33 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| 677f882d2ed8..34b9.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3e8cb31..cddc.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-01375..90b6.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..fd29.js gzip | 303 B | 303 B | ✓ |
| withRouter-b..76ae.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| buildDuration | 10s | 9.9s | -92ms |
| nodeModulesSize | 82.5 MB | 82.5 MB | -311 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| 677f882d2ed8..34b9.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3e8cb31..cddc.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-01375..90b6.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..fd29.js gzip | 303 B | 303 B | ✓ |
| withRouter-b..76ae.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| Overall change | 322 B | 322 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | bmvantunes/next.js fix/image-component-sizes-to-100vw | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.16 MB | 5.16 MB | ✓ |

</details>

</details>
Commit: 6ac7a88a76b3d4cadb2ebda6bc9ee581a33dd7c7

**Review by Timer — APPROVED** [review]

**Comment by nghiepnguyen520** [comment]
I spend 8hr on this issue with version 10.0.3. Tks GOD

**Pull request merged**